azure-pipelines.yml

```diff
@@ -142,26 +134,86 @@ stages:
     burgrName: 'qa'
     burgrType: 'qa'
     stageName: 'qa'
-    # .NET analyzers UTs, ITs, build Java and publish SC QG
+    # .NET code analysis, UTs, ITs, build Java and publish SC QG
     stageDisplayName: 'Tests:'
     stageDependencies: build
 
     jobs:
-    - job: runUnitTestsJob
-      displayName: '.NET UTs'
+    - job: runUnitTestsJobNet48
+      displayName: '.NET 4.8 UTs'
+
+      steps:
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download binaries to test'
+        inputs:
+          artifact: TestBinaries
+          targetPath: 'analyzers\tests\SonarAnalyzer.UnitTest\bin'
+
+      - powershell: |
+          . .\scripts\utils.ps1
+
+          CreateFolder("coverage")
+          CreateFolder("TestResults")
+
+          & OpenCover.Console.exe -target:"dotnet.exe" -targetargs:"test analyzers\tests\SonarAnalyzer.UnitTest\bin\$(BuildConfiguration)\net48\SonarAnalyzer.UnitTest.dll --logger trx" -returntargetcode -filter:"+[SonarAnalyzer.CFG]* +[SonarAnalyzer]* +[SonarAnalyzer.CSharp]* +[SonarAnalyzer.Utilities]* +[SonarAnalyzer.VisualBasic]*" -output:coverage/net48.xml -oldStyle
+          Test-ExitCode "ERROR: Unit tests for net48 FAILED."
+        displayName: '.Net UTs'
+
+      - task: PublishPipelineArtifact@1
```

Comment:
May be good to document the whole process in docs? Or at least provide a little bit more detailed explanation in #4796 ? Feel free to ignore that.

Reply:
I've added a small comment in the `.yaml` file. If you see unclear steps in the pipeline, please let me know, and I will try to document them.